addons/point_of_sale/views/pos_templates.xml

```diff
@@ -31,12 +31,14 @@
             var odoo = {
                 csrf_token: "<t t-esc="request.csrf_token(None)"/>",
                 session_info: <t t-raw="session_info"/>,
+                debug: "<t t-esc="request.session.debug"/>",
             };
         </script>
 
         <t t-call-assets="web.assets_common" t-css="false"/>
         <t t-call-assets="point_of_sale.pos_assets_backend" t-css="false"/>
         <t t-call-assets="point_of_sale.assets"/>
+        <t t-call="web.compiled_assets_tests"/>
```

Comment:
optional_assets_tests ?
test_assets_tests

Reply:
@qsm-odoo Maybe you can give your input here about the template name? I think that's the nomenclature used by editor and studio for wrapper template?

Reply:
Every wrapper template currently written with `_compiled` prefix, see https://github.com/odoo-dev/odoo/blame/b3c6f95645d0c487000546c5ef3eb328e09da841/addons/web_editor/views/editor.xml#L9
https://github.com/odoo-dev/odoo/blame/b3c6f95645d0c487000546c5ef3eb328e09da841/addons/website/views/website_templates.xml#L89
https://github.com/odoo-dev/enterprise/blame/254c2062b6c493531b4d17166908139fbf847848/web_studio/views/assets.xml#L103

I think it should remain like that for clearness purpose

Reply:
Yes, probably not a perfect name but at least it is consistent with the rest.

Reply:
final name: conditional_assets_test

+rename 3 other with the same convention

Reply:
I already renamed conditional_assets_test but I won't rename the other since they are not conditional but wrapper (no explicit group or if condition on it).
I don't like that and I prefered _compiled but it is done :)